server/integration/testsuite/src/test/java/org/infinispan/persistence/cluster/MyCustomCacheStore.java

```diff
@@ -2,57 +2,65 @@
 
 import org.infinispan.commons.configuration.ConfiguredBy;
 import org.infinispan.marshall.core.MarshalledEntry;
-import org.infinispan.persistence.spi.PersistenceException;
 import org.infinispan.persistence.spi.ExternalStore;
 import org.infinispan.persistence.spi.InitializationContext;
-import org.infinispan.util.logging.Log;
-import org.infinispan.util.logging.LogFactory;
+import org.infinispan.persistence.spi.PersistenceException;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.fail;
 
 /**
  * Test cache store, which never store/loads anything (= returns null for all keys).
  *
- *  @author Jakub Markos
+ * @author Jakub Markos
  */
 @ConfiguredBy(MyCustomCacheStoreConfiguration.class)
 public class MyCustomCacheStore implements ExternalStore {
 
-    private static final Log log = LogFactory.getLog(MyCustomCacheStore.class);
+   private MyCustomCacheStoreConfiguration config;
 
-    private MyCustomCacheStoreConfiguration config;
+   public MyCustomCacheStore() {
```

Comment:
Ok, on a second thought...let me switch it back to 4 spaces. 

BTW, I created JIRA for Checkstyle: https://issues.jboss.org/browse/ISPN-5683